auth-web/src/components/auth/staff/admin/AdministrativeBN.vue

```diff
@@ -0,0 +1,292 @@
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="4" class="pr-4">
+            <strong>Incorporation Number or Registration Number</strong>
+          </v-col>
+          <v-col cols="8">
+            {{ businessDetails.identifier }}
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="4" class="pr-4">
+            <strong>Business Number (BN9/BN15)</strong>
+          </v-col>
+          <v-col cols="8">
+            {{ businessDetails.taxId || '(Not Available)' }}
+          </v-col>
+        </v-row>
+      </div>
+
+      <div class="mt-10">
+        <v-alert type="error" icon="mdi-alert-circle" class="mb-0" v-if="submitBNRequestErrorMessage">
+          {{ submitBNRequestErrorMessage }}
+        </v-alert>
+        <v-alert type="success" v-if="requestQueued">
+          BN request queued.
+        </v-alert>
+
+        <v-form
+          ref="submitBNRequestForm"
+          v-on:submit.prevent="submitBNRequest"
+        >
+          <v-checkbox
+            v-model="requestCRA"
+            class="mt-0 pt-0"
+            label="Request CRA to assign a Program Account (BN15)"
+            :disabled="!canRequestNewBN()"
+          >
+          </v-checkbox>
+          <!-- Business Number -->
+          <v-text-field
+            dense
+            filled
+            persistent-hint
+            class="business-number"
+            label="Business Number (Optional)"
+            hint="First 9 digits of the CRA Business Number"
+            v-model="businessNumber"
+            v-mask="['#########']"
+            :rules="businessNumberRules"
+            :disabled="!requestCRA"
+          />
+          <v-btn
+            color="primary"
+            class="submit-btn mt-0"
+            type="submit"
+            depressed
+            :disabled="!isBNRequestFormValid()"
+            :loading="submitActive"
+          >
+            <span>Submit</span>
+          </v-btn>
+        </v-form>
+      </div>
+
+      <v-row no-gutters>
+        <v-col cols="12" class="mt-10">
+          <BNRequestManager
+            :businessIdentifier="businessDetails.identifier"
+          />
+        </v-col>
+      </v-row>
+    </template>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { BNRequest } from '@/models/request-tracker'
+import BNRequestManager from '@/components/auth/staff/admin/BNRequestManager.vue'
+import CommonUtils from '@/util/common-util'
+import { Component } from 'vue-property-decorator'
+import ConfigHelper from '@/util/config-helper'
+import { LearBusiness } from '@/models/business'
+import { SessionStorageKeys } from '@/util/constants'
+import Vue from 'vue'
+import { mask } from 'vue-the-mask'
+import { namespace } from 'vuex-class'
+
+const BusinessModule = namespace('business')
+
+@Component({
+  components: {
+    BNRequestManager
+  },
+  directives: { mask }
+})
+export default class AdministrativeBN extends Vue {
```

Comment:
Is this component exactly what you took out of AdminDashboardView.vue ? (I'm hoping to not compare it line by line.)

Reply:
Yes